socrates/lib/site/views/format.jade

```diff
@@ -0,0 +1,170 @@
+        between speakers and audience, going home alone or set schedule.
+        Doesn't matter if you're a world-class expert, world-weary salary-man,
+        a student, or have just started to program a couple of months ago.
+        Everybody has something to give, to share and to learn from others. And
+        all of that happens in a friendly, supportive and safe environment that
+        embraces and celebrates diversity.
+      p.
+        This can sound a bit unusual, but it works beautifully in practice.
+        Why? Because so many amazing people are at the same place, focusing on
+        each other. By the end of the conference you'll know by name a few
+        dozens of new faces. You'll have made friends, business connections if
+        you want them, maybe you'll find a mentor, or a kindred spirit for that
+        interesting project you want to work on. Anything can happen. And is
+        bound to happen.
+      p
+        | And
+        b  that
+        |  is what SoCraTes is really all about.
+
+      h3 Schedule
+      p.
+        So now that you know what SoCraTes actually is, here are a few more
+        details about the schedule.
+      dl
+
+        dt Thursday
+        dd.
+          The conference starts at 5pm on Thursday, August 25th 2016.
+          This is the time to get to know new faces or meet old friends, have
+          some drinks at the bar, explore the surrounding area etc.
+          T-Shirts are being distributed.
+        dd.
+          Starting around 6pm the hotel will serve a buffet so that even
+          people who arrived a little later will get something to eat.
+        dd Later around 7.30pm we will officially start the conference with a #{''}
+          +extlink('https://en.wikipedia.org/wiki/The_World_Cafe', 'World Café')
+          | , to help kickstart everyone into a great conference experience.
+
+        dt Friday
+        dd Friday is completely reserved for a full-day #{''}
+          +extlink('http://agilecoachcamp.org/tiki-index.php?page=OpenSpace', 'Open Space')
+          | .
+        dd
+          | The marketplace starts at 9am, facilitated by the outstanding facilitator #{''}
+          +extlink('http://blog.connexxo.com/', 'Pierluigi Pugliese')
+          |.
+          | The first group of sessions will start around 10.30am.
+          | The last session timeslot ends at 6pm, which is also
+          | the time we all meet again in the marketplace room for a joint
+          | retrospective of the day.
+        dd
+          | Around 7pm dinner is available for around 2h, so everyone can start
+          | into the evening at their own pace. Of course there's lots of room for
+          | shared evening activities like board games, making music, hacking
+          | or just talking and having fun. Take a look at #{''}
+          a(href='/wiki/' + (currentYear - 1) + '/evening-activities') last year's activities
```

Comment:
renders as "/wiki/edit/NaN/evening-activities" -> one more reason to not just create new pages! :)